homeassistant/components/lightwave/__init__.py

```diff
@@ -50,4 +76,19 @@
             async_load_platform(hass, "switch", DOMAIN, switches, config)
         )
 
+    trv = config[DOMAIN][CONF_TRV]
+    trvs = trv[CONF_TRVS]
+    if trv:
+        proxy_ip = trv[CONF_PROXY_IP]
+        proxy_port = trv[CONF_PROXY_PORT]
+        hass.data[LIGHTWAVE_TRV_PROXY] = proxy_ip
+        hass.data[LIGHTWAVE_TRV_PROXY_PORT] = proxy_port
```

Comment:
Do we need to save this in `hass.data`? Since it is not set again used in any other place?

Reply:
You are right, this is no longer needed - a hangover from a previous requested change.
It's gone!